src/components/App/styles.module.scss

```diff
@@ -3,12 +3,14 @@
 }
 
 .header {
+  align-items: center;
   background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
+  color: white;
```

Comment:
The `color` should probably in `index.css`, inside `body`, no? Unless this can be specified with a bootstrap SCSS variable?

Reply:
There is a bootstrap variable for this. I've updated the patch to use that.